Website/DesktopModules/Hotcakes/web.config

```diff
@@ -42,6 +42,14 @@
         <assemblyIdentity name="System.Net.Http.Formatting" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-5.2.3.0" newVersion="5.2.3.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Bcl.AsyncInterfaces" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
```

Comment:
This appears to be a new dependency. Was this required by Stripe or something else?

Reply:
In the last code review, I commented that the stripe solution was hosted in the Hotcakes project. In the current version, the Stripe.dll file comes from the references folder. I assume they compiled Stripe's solution to generate this solution with all its dependencies in a single dll file. With the NuGet approach, all dependencies must be added. This must be the reason why the new references appear. I have checked these packages on the nuget.org site and they are stripe.net dependencies